scripts/plotting_tools.py

```diff
@@ -0,0 +1,329 @@
+import os
+from os.path import join
+from glob import glob
+
+import pandas as pd
+import xarray as xr
+import numpy as np
+
+from matplotlib import pyplot as plt
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+
+import logging
+
+ptypes = ['snow', 'rain', 'frzr', 'icep']
+
+def get_tle_files(base_path, valid_time, n_members=1, min_lead_time=1, init_step=1):
+    '''
+    Get ptype files at a given valid time.
```

Comment:
I would remove the `init_step` and the `min_lead_time` arguments all together since they're not used.